omero/sysadmins/server-ldap.txt

```diff
@@ -139,7 +139,11 @@ the value of ``new_user_group``, which can have several different values:
   group is mapped via ``group_mapping``. You can prefix this value with
   ``filtered_`` to have the ``group_filter`` applied to the attribute values,
   i.e. ``:filtered_dn_attribute:memberOf`` will mean that only the values of
-  memberOf which match ``group_filter`` will be considered.
+  memberOf which match ``group_filter`` will be considered. Note that if an
+  attribute specified in ``omero.ldap.group_mapping`` does not constitute a
```

Comment:
I would use `` :property:`omero.ldap.group_mapping` `` to add a reference to the configuration properties page.